test/core/elem.wast

```diff
@@ -379,3 +379,18 @@
 (assert_return (invoke $module1 "call-7") (i32.const 67))
 (assert_return (invoke $module1 "call-8") (i32.const 69))
 (assert_return (invoke $module1 "call-9") (i32.const 70))
+
+;; Duplicate table identifiers
```

Comment:
These have nothing to do with elem, so should go to a new table.wast test (which for some reason we are lacking).

Reply:
Ah good point, done now! Wasn't sure if that was ok to add here :)

Also the tests should be updated to actually pass now